README.md

```diff
@@ -1,19 +1,30 @@
-# CarbonData
+<!--
```

Comment:
Please update `git clone https://github.com/HuaweiBigData/carbondata.git` to 'https://github.com/apache/incubator-carbondata.git'